### DISCUSSION TITLE
hello, MDAnalysis. i am myles

### DISCUSSION THREAD

**Discussion by mylestande** [MDAnalysis/mdanalysis#4491] [Introductions]
good day to all! i am called myles, and i am excited about contributing to MDAnalysis under Jen's guidance, and ultimately joining the project :wave: 

no, really, i am excited to be here. just look at the smiley emoji :grin:

**Comment by hmacdope** [comment]
Hi Miles! Welcome.

**Comment by adetutudeborah** [comment]
Welcome Myles! We are glad to have you join the community.

**Comment by ummyhajj** [comment]
You are welcome @myles-bilong
